packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,223 @@
+  }, [navigate, email]);
+
+  const onSubmit = useCallback(
+    async (email) => {
+      try {
+        // TODO confirm how to pass entrypoint_variation
+        logViewEvent(settingsViewName, 'reset-password.submit', {
+          entrypoint_variation: 'react',
+        });
+        // TODO password reset method - currently only console logs
+        await account.resetPassword(email);
+        logViewEvent(settingsViewName, 'reset-password.success');
+        navigateToConfirmPwReset();
+      } catch (e) {
+        logViewEvent(settingsViewName, 'reset-password.fail');
+        // TODO error handling - code below copied from content-server
+        /*
+          // clear oauth session
+          Session.clear('oauth');
+          if (AuthErrors.is(err, 'UNKNOWN_ACCOUNT')) {
+            err.forceMessage = t('Unknown account. <a href="/signup">Sign up</a>');
+            return this.unsafeDisplayError(err);
+          } else if (AuthErrors.is(err, 'USER_CANCELED_LOGIN')) {
+            this.logEvent('login.canceled');
+            // if user canceled login, just stop
+            return;
+          }
+          // re-throw error, it will be handled at a lower level.
+          throw err;
+        */
+        // TODO display error tooltip if email not associated with an account
+        // error message is passed to InputText > Tooltip
+        // by setting the error message with setEmailErrorText
+        // error message returned from account.resetPassword method or defined here?
+        setEmailErrorText('Unknown account');
+        // TODO error handling for other errors - w/ alertBar?
+        alertBar.error(
+          l10n.getString(
+            'pw-reset-error',
+            null,
+            'Sorry, there was a problem resetting your password'
+          )
+        );
+      }
+    },
+    [l10n, navigateToConfirmPwReset, alertBar, account]
+  );
+
+  return (
+    <>
+      <header data-testid="reset-password-header" className="mb-4">
+        <Localized id="pw-reset-header">
+          <h1
+            id="fxa-reset-password-header"
+            data-testid="reset-password-header-text"
+            className="text-xl mb-2"
+          >
+            Reset password
+          </h1>
+        </Localized>
+        <Localized id="pw-reset-subheader">
+          {/* TODO service name should be localized - currently remains in English */}
+          <h2 data-testid="reset-password-subheader">
+            {`to continue to ${displayedServiceName}`}
+          </h2>
+        </Localized>
+      </header>
+
+      <section>
+        <form
+          noValidate
+          className="flex flex-col gap-4"
+          onSubmit={handleSubmit(onSubmit)}
+          data-testid="reset-password-form"
+        >
+          <Localized id="pw-reset-warning">
+            <p className="my-4 text-sm" data-testid="reset-password-warning">
+              {/* TODO This is a BIG block of text. Could benefit from a UX review */}
```

Comment:
I've always wondered about this, seems like we can have a dedicated page or something to let our users know what happens? Then again, what are the odds they click the link 😅

Reply:
Yeah, it doesn't seem like the friendliest/clearest UX. I can't help but wonder if improving this would help reduce the instances of data loss from password reset (could potentially help with that data loss prevention OKR for 2023?)